backend/src/main/java/de/neuefische/backend/service/ItemService.java

```diff
@@ -0,0 +1,37 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.model.Item;
+import de.neuefische.backend.repository.ItemRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+
+@Service
+@RequiredArgsConstructor
+public class ItemService {
+    private final ItemRepository itemRepository;
+
+    public List<Item> getAll() {
+        return itemRepository.findAll();
+    }
+
+    public Item getById(String id) throws Exception {
+        return itemRepository.findById(id).orElseThrow(() -> new Exception("Item with id " + id + " is nonexistent."));
+    }
```

Comment:
You could absolutely make yourself your own custom exception for that. It's fairly simple and looks real cool! Give me a shout if you want me to hook you up with one

Reply:
Ja, warum nicht! Kann ich mir morgen gern mal anschauen!